dynamic_programming/longest_common_substring.py

```diff
@@ -0,0 +1,61 @@
+"""
+Longest Common Substring Problem Statement: Given two sequences, find the length
+of longest common substring present in both of them. A substring is
+necessarily continuous.
+Example: "abcdef" and "xabded" have two longest common substrings, "ab" or "de".
+Therefore, the length is 2.
+"""
+
+
+def longest_common_substring(text1: str, text2: str) -> int:
```

Comment:
Given that converting a string to its length is trivial in Python, please return the longest substring instead of its length.  If multiple strings are the same length then return one of them.
Hint: `max("abcde", "wxyz", key=len)`
```suggestion
def longest_common_substring(text1: str, text2: str) -> str:
```